src/azul/terraform.py

```diff
@@ -755,10 +756,17 @@ def tf_config(self, app_name):
                 '${aws_vpc_endpoint.%s.id}' % app_name
             ]
 
+        assert 'aws_lambda_alias' not in resources
+        resources['aws_lambda_alias'] = dict()
```

Comment:
```suggestion
        resources['aws_lambda_alias'] = {}
```

But the logical thing to do would be to populate a variable first before injecting its value into the larger structure.

Reply:
Found it cleaner to implement as a dictionary comprehension like `resources['aws_cloudwatch_log_group']`